src/js/components/LogDetails/index.tsx

```diff
@@ -8,22 +8,21 @@ import LogDetails from "../../state/LogDetails"
 import NoSelection from "./NoSelection"
 import UidPanel from "./UidPanel"
 import menu from "../../electron/menu"
-import {createZeekLog} from "../../brim/zeekLog"
+import {Correlation} from "../../correlation/models/Correlation"
 
 export default function LogDetailComponent() {
-  const log = useSelector(LogDetails.build)
-  const zeek = createZeekLog(log)
-  console.log(log)
-  if (!log) return <NoSelection />
+  const record = useSelector(LogDetails.build)
+  if (!record) return <NoSelection />
 
+  const correlation = new Correlation(record)
```

Comment:
I may or may not do away with this class in the future. It just basically knows to to find a uid and community_id from a record.

In the future we may use the ZeekEvent and SuricataEvent classes (which we store in the ppl directory), that wrap a record and have knowledge of the Zeek and Suricata fields.